src/Jellyfin.Extensions/Json/Converters/JsonDelimitedArrayConverter.cs

```diff
@@ -75,7 +66,39 @@ protected JsonDelimitedArrayConverter()
         /// <inheritdoc />
         public override void Write(Utf8JsonWriter writer, T[]? value, JsonSerializerOptions options)
         {
-            throw new NotImplementedException();
+            if (value is not null)
+            {
+                writer.WriteStartArray();
+                if (value.Length > 0)
+                {
+                    var toWrite = value.Length - 1;
+                    foreach (var it in value)
+                    {
+                        var wrote = false;
+                        if (it is not null)
+                        {
+                            writer.WriteStringValue(it.ToString());
+                            wrote = true;
+                        }
+
+                        if (toWrite > 0)
+                        {
+                            if (wrote)
+                            {
+                                writer.WriteStringValue(Delimiter.ToString());
```

Comment:
Not sure about the perf impact of doing `ToString` everytime :thinking: 
For now this should be fine but I'll do some benchmarks when I've got time.